php-nginx/php.ini

```diff
@@ -586,7 +586,7 @@ html_errors = Off
 ; Example:
 ;error_log = php_errors.log
 ; Log errors to syslog (Event Log on Windows).
-error_log = /var/log/app_engine/php-error.log
+error_log = /var/log/app_engine/application.log
```

Comment:
OK. However, I don't see anything in that file (php-error.log or application.log) when I emit text to stdout/stderr.